dao/src/test/java/io/syndesis/dao/DataManagerTest.java

```diff
@@ -51,9 +56,8 @@ public void setup() {
     @Test
     public void getConnectors() {
         ListResult<Connector> connectors = dataManager.fetchAll(Connector.class);
-        for (Connector connector : connectors.getItems()) {
-            System.out.print(connector.getId().get() + ",");
-        }
+        assertThat(connectors.getItems().stream().map(Connector::getId).map(Optional::get))
+            .containsExactly("gmail", "github", "ftp", "facebook", "linkedin", "salesforce", "timer", "jms", "day-trade", "twitter", "servicenow", "http", "trade-insight");
```

Comment:
Aha and the list comes from `deployment.json`, so yes this will fail every time that changes.